# *S.* Napoli

!!! info "At a Glance"

    | | |
    |---|---|
    | **Antigenic Formula** | `1,9,12:l,z13:e,n,x` |
    | **Serogroup** | O:9 (D1) |
    | **NCBI Pathogen Detection** | [View isolates](<https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Napoli%22)>) |

## Background Information

*Salmonella enterica* subsp. *enterica* serovar Napoli (antigenic formula 1,9,12:l,z<sub>13</sub>:e,n,x) is a serovar of the O:9 (D<sub>1</sub>) serogroup. In 2006, serovar Napoli ranked [22nd](https://pubmed.ncbi.nlm.nih.gov/19534593/) in the Enter-net *Salmonella* database, accounting for 295 cases (0.28%) out of 105,635 reported across 29 European countries. Between 2000 and 2006, Europe saw a [140%](https://www.liebertpub.com/doi/10.1089/fpd.2008.0206) rise in serovar Napoli infections, with over 87% of cases originating in France, Italy, and Switzerland. This serovar has been an emerging serovar in these three countries, particularly in Italy, where it is the [fourth most frequent](https://www.escaide.eu/sites/default/files/documents/ESCAIDE-2012-abstract-book.pdf) *Salmonella* serovar, responsible for [2-4%](https://www.scielosp.org/article/aiss/2014.v50n1/96-98/) of human infections. Additionally, from 2010 to 2014, serovar Napoli was the leading cause of invasive salmonellosis, affecting [40 out of 687](https://www.liebertpub.com/doi/abs/10.1089/fpd.2016.2206) cases in Italy—a notably higher invasive rate compared to other nontyphoidal serotypes. Although serovar Napoli is a nontyphodial *Salmonella* serovar some of its symptoms, including a prolonged incubation period (7–14 days), [high fever, bacteremia, and elevated rates of hospitalizations](https://www.liebertpub.com/doi/10.1089/fpd.2016.2206?doi=10.1089/fpd.2016.2206), are similar to those caused by typhoidal *Salmonella*. Serovar Napoli has also been frequently isolated from [rucola (rocket salad)](https://pubmed.ncbi.nlm.nih.gov/21561382/).

## Genetic Characteristics

Serovar Napoli has been found to be [monophyletic](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10). An [Italian study](https://www.liebertpub.com/doi/10.1089/fpd.2016.2206?doi=10.1089%2Ffpd.2016.2206) used genomic and phylogenetic approaches and found that an invasive strain of serovar Napoli was part of the Typhi subclade within clade A, with Paratyphi A as its closest serovar. This isolate was found to have the typhoid-associated genes *hlyE* and *taiA* (SPI-18), *cdtB, pltA,* and *pltB* (CdtB-Islet), as well as the GICT18/1 islet. Another [Italian study](https://www.liebertpub.com/doi/10.1089/fpd.2010.0833?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed) analyzed 112 serovar Napoli isolates and revealed that all tested isolates were susceptible to the examined antimicrobials, with SPI-associated genes (*ssaQ*, *mgtC*, *spi_4D*, and *sopB*) present in 75–100% of isolates, while *avrA* was rare (detected in only one human and four environmental isolates). The phage-related *sopE1* appeared in 93% of strains, whereas *sodC1* and *gipA* were limited to four and two environmental isolates, respectively. The fimbrial gene *bcfC* was universal in animal/food strains but less frequent in environmental (71.4%) and human (46.8%) isolates. Ten virulence profiles (VPs) were identified, with VP1–3 encompassing 99 of 112 isolates. PFGE analysis grouped 103 of 111 isolates into four major and three minor clusters (two isolates were unrelated), with dominant clusters strongly correlating with VP1–3.

[Mastrorilli et al.](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6588-y) further corroborated the genomic relatedness of serovar Napoli to typhoidal serovars (*Salmonella* Typhi and Paratyphi A), with two distinct subclades evident, including the biphyletically distributed ST-474. All sequence types (STs) were distributed across isolation sources and years. Compared to other non-typhoidal serovars, serovar Napoli harbors fewer SPIs and ARGs and rarely acquires plasmids. Notably, their study documented the second known instance of an extended-spectrum β-lactamase (ESBL)-producing serovar Napoli isolate and the first reported cases of MDR serovar Napoli isolates, all of human origin.

## Animal Reservoir

Unknown, [poultry](https://pmc.ncbi.nlm.nih.gov/articles/PMC8276318/#:~:text=So%20far%2C%20it%20has%20been,undoubted%20benefit%20at%20community%20level.) and [wild boars](https://link.springer.com/article/10.1007/s10393-023-01625-y) may be potential reservoirs for serovar Napoli. It was also isolated from [wild birds](https://www.scielosp.org/article/aiss/2014.v50n1/96-98/) but [uncommonly from cattle](https://efsa.onlinelibrary.wiley.com/doi/pdf/10.2903/sp.efsa.2022.EN-7687).

## Geographical Distribution

Serovar Napoli has been reported in [US, Canada, Australia, and Europe, mainly in Italy.](<https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Napoli%22)>)

## Human/Animal Outbreaks

| Year | Location | Associated source | Number of cases |
| --- | --- | --- | --- |
| 2023 | Norway | [Not identified](https://www.foodsafetynews.com/2023/12/salmonella-outbreak-over-in-norway-but-source-not-found/) | 7 |
| 2014 | Italy | [Ham](https://www.liebertpub.com/doi/full/10.1089/fpd.2015.2091) | 47 |
| 2011 | Italy | [Water](https://www.escaide.eu/sites/default/files/documents/ESCAIDE-2012-abstract-book.pdf) | 10 |
| 1982 | England and Wales | [Chocolate bars](https://pubmed.ncbi.nlm.nih.gov/6131266/) | 245 |

Animal outbreaks:

| Year | Location | Breed | Number of cases |
| --- | --- | --- | --- |
| 2022-2023 | Italy | [Holstein Friesian](https://pmc.ncbi.nlm.nih.gov/articles/PMC11718894/) | 4<sup>1</sup> |

<sup>1</sup> The herd consisted of 310 cows. There were four animals infected: two calves and two milking cows.

## Border Rejections

There have been no recent border rejections linked to this serovar.

## Recalls

There have been no recent recalls linked to this serovar.

## References

1. [https://pubmed.ncbi.nlm.nih.gov/19534593/](https://pubmed.ncbi.nlm.nih.gov/19534593/)
2. [https://www.liebertpub.com/doi/10.1089/fpd.2008.0206](https://www.liebertpub.com/doi/10.1089/fpd.2008.0206)
3. [https://www.escaide.eu/sites/default/files/documents/ESCAIDE-2012-abstract-book.pdf](https://www.escaide.eu/sites/default/files/documents/ESCAIDE-2012-abstract-book.pdf)
4. [https://www.scielosp.org/article/aiss/2014.v50n1/96-98/](https://www.scielosp.org/article/aiss/2014.v50n1/96-98/)
5. [https://www.liebertpub.com/doi/abs/10.1089/fpd.2016.2206](https://www.liebertpub.com/doi/abs/10.1089/fpd.2016.2206)
6. [https://www.liebertpub.com/doi/10.1089/fpd.2010.0833?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed](https://www.liebertpub.com/doi/10.1089/fpd.2010.0833?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed)
7. [https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6588-y](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6588-y)
8. [https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10)
9. [https://pubmed.ncbi.nlm.nih.gov/21561382/](https://pubmed.ncbi.nlm.nih.gov/21561382/)
10. [https://pmc.ncbi.nlm.nih.gov/articles/PMC8276318/#:~:text=So%20far%2C%20it%20has%20been,undoubted%20benefit%20at%20community%20level](https://pmc.ncbi.nlm.nih.gov/articles/PMC8276318/#:~:text=So%20far%2C%20it%20has%20been,undoubted%20benefit%20at%20community%20level).
11. [https://link.springer.com/article/10.1007/s10393-023-01625-y](https://link.springer.com/article/10.1007/s10393-023-01625-y)
12. [https://efsa.onlinelibrary.wiley.com/doi/pdf/10.2903/sp.efsa.2022.EN-7687](https://efsa.onlinelibrary.wiley.com/doi/pdf/10.2903/sp.efsa.2022.EN-7687)
13. [https://www.foodsafetynews.com/2023/12/salmonella-outbreak-over-in-norway-but-source-not-found/](https://www.foodsafetynews.com/2023/12/salmonella-outbreak-over-in-norway-but-source-not-found/)
14. [https://www.liebertpub.com/doi/full/10.1089/fpd.2015.2091](https://www.liebertpub.com/doi/full/10.1089/fpd.2015.2091)
15. [https://pubmed.ncbi.nlm.nih.gov/6131266/](https://pubmed.ncbi.nlm.nih.gov/6131266/)
16. [https://pmc.ncbi.nlm.nih.gov/articles/PMC11718894/](https://pmc.ncbi.nlm.nih.gov/articles/PMC11718894/)

