# *S.* Livingstone


## Background Information

 *Salmonella enterica* subsp. *enterica* serovar Livingstone (antigenic formula 6,7,14:d:l,w) is a serovar of the O:7 (C1) serogroup. This serovar was first isolated in [1951](https://onlinelibrary.wiley.com/doi/10.1002/path.1700660136) from human feces in UK. From [1999 to 2003](https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/), there was a huge surge in the isolation of serovar Livingstone from human salmonellosis infections and animal-based food products in Tunisia. During [1989-1991](https://pubmed.ncbi.nlm.nih.gov/8107062/), serovar Livingstone was the third most prevalent *Salmonella* serotype isolated from human salmonellosis infections in the Tayside area of Scotland.

NCBI PD Serovar Link for Serovar Livingstone: [https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Livingstone%22)](https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Livingstone%22))

## Genetic Characteristics

 Serovar Livingstone has been found to be [polyphyletic](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10) with four lineages identified and one stand-alone singleton that does not cluster with any other *Salmonella* Livingstone isolates. In a [nosocomial outbreak associated with serovar Livingstone in Tunisia](https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/), all outbreak strains (n=16) exhibited resistance to ceftriaxone and ceftazidime, mediated by the production of an extended-spectrum =CE=B2-lactamase (ESBL). Additionally, they showed resistance to multiple aminoglycosides (kanamycin, tobramycin, netilmicin, gentamicin, and amikacin) as well as sulfamethoxazole-trimethoprim. The ESBL resistance was encoded on a 40-kb conjugative plasmid. Genetic analysis revealed the presence of the mobile insertion sequence IS*Ecp1* upstream of *blaCTX-M-27*, positioned similarly to its arrangement in *blaCTX-M-14*. A novel trimethoprim resistance gene, *dfrA21*, was identified on a 90-kb plasmid. This gene was integrated as a single resistance cassette within a class I integron. [Li et al.](https://pmc.ncbi.nlm.nih.gov/articles/PMC9641423/) identified five clades of serovar Livingstone, with evidence suggesting that the global animal feed trade likely facilitated their introduction into China. Among these, Clade-5-I-a/b—the predominant lineage in China—demonstrated invasiveness in murine, avian (chicken), and zebrafish infection models. Antimicrobial susceptibility testing indicated that the vast majority (>96%) of Chinese Livingstone isolates exhibited multidrug resistance. In a [comparative study](https://jidc.org/index.php/journal/article/view/25116661), all Tunisian isolates exhibited resistance to amoxicillin, amoxicillin-clavulanic acid, ticarcillin, cefalotin, gentamicin, and kanamycin, as well as to third-generation cephalosporins (cefotaxime and ceftazidime). In contrast, Belgian isolates remained susceptible to all tested antibiotics. Genotypically, the Tunisian isolates were homogeneous, belonging exclusively to sequence type ST543, whereas the Belgian isolates exhibited greater diversity with eight strains typed as ST543, two as ST638, and one as ST457. All isolates including them from Tunisia and Belgium carried five chromosomal genes (*agfA*, *hin/H2*, *iroB*, *phoP/Q*, and *slyA*) but lacked the plasmid-encoded *spvA* and *spvC*.

## Animal Reservoir

 [Poultry](https://www.microbiologyresearch.org/content/journal/jmm/10.1099/00222615-40-2-134?crawler=true) is likely to be a potential reservoir for serovar Livingstone.

## Geographical Distribution

 Serovar Livingstone has been reported [worldwide](https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/).

## Human/Animal Outbreaks

| Year | Location | Associated source | Number of cases |
| --- | --- | --- | --- |
| 2023-2024 | US: Utah | [Restaurant-related (no common source was identified)](https://pubmed.ncbi.nlm.nih.gov/38870481/) | 11 |
| 2002 | Tunisia | [Nosocomial](https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/) | 16 |
| 2001 | Norway, Sweden | [Processed fish products](https://www.jstor.org/stable/3865763?seq=1) | 44 in Norway, 16 In Sweden |
| 1989-1991 | Scotland | [Not identified](https://www.microbiologyresearch.org/content/journal/jmm/10.1099/00222615-40-2-139) | 71 |

## Human/Animal Outbreaks

| Year | Location | Breed | Number of cases |
| --- | --- | --- | --- |
| 1980 | UK | [CBA/ca mice](https://journals.sagepub.com/doi/10.1258/002367781780893812) | 60^1^ |

^1^ Only mice under 3 weeks old exhibited symptoms of enteritis. They were treated orally with tetracycline, 10 g/L drinking water for 10 days.

## Border Rejections

| **Year** | **Exporting country** | **Importing country** | **Associated source** | **Product category** |
| --- | --- | --- | --- | --- |
| 2020 | US | Greece | [Fish meal](https://webgate.ec.europa.eu/rasff-window/screen/notification/429239) | Feed materials |

## Recalls

| Year | Location | Recalled food | Type |
| --- | --- | --- | --- |
| 2024 | Slovakia | [Rucola from Italy](https://webgate.ec.europa.eu/rasff-window/screen/notification/722139) | Fruits and vegetables |
| 2020 | Poland | [Hulled sesame seeds](https://webgate.ec.europa.eu/rasff-window/screen/notification/433397)^1^ | Nuts, nut products and seeds |

^1^ *Salmonella *Amsterdam and *Salmonella* Livingstone were found in sesame seeds from India, packed in Poland.

## Relevant Links

1. [https://onlinelibrary.wiley.com/doi/10.1002/path.1700660136](https://onlinelibrary.wiley.com/doi/10.1002/path.1700660136)
2. [https://pmc.ncbi.nlm.nih.gov/articles/PMC9641423/](https://pmc.ncbi.nlm.nih.gov/articles/PMC9641423/)
3. [https://jidc.org/index.php/journal/article/view/25116661](https://jidc.org/index.php/journal/article/view/25116661)
4. [https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full#h10)
5. [https://pubmed.ncbi.nlm.nih.gov/38870481/](https://pubmed.ncbi.nlm.nih.gov/38870481/)
6. [https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/](https://pmc.ncbi.nlm.nih.gov/articles/PMC1081247/)
7. [https://www.microbiologyresearch.org/content/journal/jmm/10.1099/00222615-40-2-139](https://www.microbiologyresearch.org/content/journal/jmm/10.1099/00222615-40-2-139)
8. [https://www.jstor.org/stable/3865763?seq=1](https://www.jstor.org/stable/3865763?seq=1)
9. [https://journals.sagepub.com/doi/10.1258/002367781780893812](https://journals.sagepub.com/doi/10.1258/002367781780893812)
10. [https://webgate.ec.europa.eu/rasff-window/screen/notification/429239](https://webgate.ec.europa.eu/rasff-window/screen/notification/429239)
11. [https://webgate.ec.europa.eu/rasff-window/screen/notification/722139](https://webgate.ec.europa.eu/rasff-window/screen/notification/722139)
12. [https://webgate.ec.europa.eu/rasff-window/screen/notification/433397](https://webgate.ec.europa.eu/rasff-window/screen/notification/433397)
