# *S.* Orion


!!! info "At a Glance"

    | | |
    |---|---|
    | Serogroup | O:3,10 (E1) |
    | NCBI Pathogen Detection | [View isolates](https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Orion%22)) {:target="_blank"} |

## Background Information

 *Salmonella enterica* subsp. *enterica* serovar Orion (antigenic formula 3,{10}{15}{15,34}:y:1,5) is a serovar of the O:3,10 (E1) serogroup. This serovar was first identified in [1945](https://journals.asm.org/doi/10.1128/jb.50.5.577-578.1945?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub++0pubmed) from a rectal swab sample taken from a man in US. [McAuley et al.](https://www.sciencedirect.com/science/article/pii/S0362028X2209593X) found that serovar Orion was the most prevalent *Salmonella* serovar and was potentially endemic to seven dairy farms in Australia.

%20AND%20computed_types:(%22serotype=Orion%22))

## Genetic Characteristics

 Serovar Orion has been found to be [polyphyletic](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full) with two lineages identified and one stand-alone singleton that does not cluster with any other *Salmonella* Orion isolates. Serovar Orion, specifically its variant 15^+^,34^+^, was classified as belonging to the [low-invasive](https://journals.asm.org/doi/10.1128/aem.02931-14) groups of *Salmonella* serovars. [Agga et al.](https://pubmed.ncbi.nlm.nih.gov/34520249/) isolated a serovar Orion strain from mink feces, which carried copper and silver resistance gene clusters (e.g., *golS*, *pcoA*, and *silA*) and multidrug resistance genes (i.e., *mdsA* and *mdsB*). Similarly,** **[Hasegawa](https://onlinelibrary.wiley.com/doi/10.1111/zph.13140) et al. also found that** **all 324 serovar Orion genomes analyzed contained the efflux pump genes *mdsA*/*mdsB* and gold tolerance genes *golS*/*golT*, while other resistance genes ( aminoglycosides, =CE=B2-lactams, quinolones [via chromosomal mutations], colistin, lincosamides, macrolides, phenicols, sulfonamides, trimethoprim, tetracyclines, and disinfectants) were present at lower frequencies (0.3%–5.9%), with 17 genomes (5.2%) from Canada, the UK, US, and Tanzania showing multidrug-resistant potential. Metal tolerance genes for arsenic, copper, mercury, silver, and tellurium appeared in 0.3%–35.5% of isolates, and the plasmid Col(MGD2) was most prevalent (15.4%). Virulence profiling identified genes for adherence, macrophage induction, magnesium uptake, regulation, serum resistance, stress adaptation, type III secretion systems, and six SPIs (1–5, 9, 12–14, C63PI). Typing revealed 89.2% of isolates belonged to ST639.

[Ragione et al.](https://www.sciencedirect.com/science/article/pii/S1438422101800114?via%3Dihub) found that serovar Orion var. 15^+^ demonstrated comparable adherence and invasion capabilities to *Salmonella* Enteritidis in HEp-2 cell cultures. However, in one-day-old and seven-day-old chick models, serovar Orion var. 15^+ ^exhibited significantly lower virulence. While *Salmonella* Enteritidis colonized 100% of liver and spleen samples within 24 hours at a given dose, serovar Orion var. 15^+^ only colonized 25% of these organs. In another trial, var. 15^+^ failed to colonize any livers or spleens by 24 hours, whereas *Salmonella* Enteritidis successfully colonized 50% of organs at the same dose. Among the three tested serovar Orion var. 15^+^ isolates (F3720, B1, and B7), strain F3720 showed significantly greater invasiveness, colonizing more internal organs than the others. Interestingly, F3720 was also shed in lower quantities compared to B1 and B7, suggesting a potential inverse relationship between deep tissue colonization and gut persistence.

## Animal Reservoir

 Unknown, but according to the metadata from NCBI PD, serovar Orion was isolated from [pigs, cattle, chicken and other animals](https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Orion%22)).

## Geographical Distribution

 According to the metadata from NCBI PD (accessed 05/13/2025), serovar Orion has been reported [worldwide but is predominantly found in US](https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Orion%22)) (277 of 512 serovar Orion isolates were collected in US).

## Human/Animal Outbreaks

| Year | Location | Associated source | Number of cases |
| --- | --- | --- | --- |
| 2019-2021 | EU: multi-country | [Halva or tahini from Syria](https://www.ecdc.europa.eu/en/publications-data/rapid-outbreak-assessment-multi-country-outbreak-multiple-salmonella-enterica)^1^ | 121 |

^1^ *Salmonella* Amsterdam, Havana, Kintambo, Mbandaka, Orion, and Senftenberg were involved in this outbreak. The case number here only reflects the total number of people who got infected, not just those infected by serovar Orion.

## Border Rejections

| Year | Exporting country | Importing country | Associated source | Product category |
| --- | --- | --- | --- | --- |
| 2022 | Uganda | Germany | [Organic sesame seeds](https://webgate.ec.europa.eu/rasff-window/screen/notification/575409) | Nuts, nut products and seeds |

## Recalls

| Year | Location | Recalled food | Type |
| --- | --- | --- | --- |
| 2022 | Norway | [Halva from Syria](https://webgate.ec.europa.eu/rasff-window/screen/notification/542613)^1^ | Confectionery |
| 2021 | Germany | [Tahini halva with pistachios from Syria](https://webgate.ec.europa.eu/rasff-window/screen/notification/509031)^2^ | Confectionery |

^1^ *Salmonella* Senftenberg and Orion were found.

^2^ This recall was caused by a multi-country outbreak described above. *Salmonella* Orion and group E serovars were found.

## References

1. [https://journals.asm.org/doi/10.1128/jb.50.5.577-578.1945?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub++0pubmed](https://journals.asm.org/doi/10.1128/jb.50.5.577-578.1945?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub++0pubmed)
2. [https://www.sciencedirect.com/science/article/pii/S0362028X2209593X](https://www.sciencedirect.com/science/article/pii/S0362028X2209593X)
3. [https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full)
4. [https://journals.asm.org/doi/10.1128/aem.02931-14](https://journals.asm.org/doi/10.1128/aem.02931-14)
5. [https://pubmed.ncbi.nlm.nih.gov/34520249/](https://pubmed.ncbi.nlm.nih.gov/34520249/)
6. [https://onlinelibrary.wiley.com/doi/10.1111/zph.13140](https://onlinelibrary.wiley.com/doi/10.1111/zph.13140)
7. [https://www.sciencedirect.com/science/article/pii/S1438422101800114?via%3Dihub](https://www.sciencedirect.com/science/article/pii/S1438422101800114?via%3Dihub)
8. [https://www.ecdc.europa.eu/en/publications-data/rapid-outbreak-assessment-multi-country-outbreak-multiple-salmonella-enterica](https://www.ecdc.europa.eu/en/publications-data/rapid-outbreak-assessment-multi-country-outbreak-multiple-salmonella-enterica)
9. [https://webgate.ec.europa.eu/rasff-window/screen/notification/575409](https://webgate.ec.europa.eu/rasff-window/screen/notification/575409)
10. [https://webgate.ec.europa.eu/rasff-window/screen/notification/542613](https://webgate.ec.europa.eu/rasff-window/screen/notification/542613)
11. [https://webgate.ec.europa.eu/rasff-window/screen/notification/509031](https://webgate.ec.europa.eu/rasff-window/screen/notification/509031)
