# *S.* Heidelberg

!!! info "At a Glance"

    | | |
    |---|---|
    | **Antigenic Formula** | `1,4,[5],12:r:1,2` |
    | **Serogroup** | O:4 (B) |
    | **NCBI Pathogen Detection** | [View isolates](https://www.ncbi.nlm.nih.gov/pathogens/isolates/) |


## Background Information

*Salmonella enterica* subsp. *enterica* serovar Heidelberg (antigenic formula 1,4,[5],12:​r:1,2) is a serovar of the O:4 (B) serogroup. *Salmonella* Heidelberg was first identified in [1933 in Heidelberg, Germany](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=About%20a%20new%20type%20of%20bacteria%20form%20the%20paratyphoid%20enteritis%20group&author=VH%20Habbs&publication_year=1933&pages=367-374&). It was later detected in [the United States for the first time in 1954](https://www.jstor.org/stable/1588278?casa_token=sTneApEGY84AAAAA%3AxCcrraoy6aPu-M6ejq3MtxjBB1MMgP_Z6YwUi4_cVoYN9dHewNuMwxG9EeHAe6ua_dq1zAdT8vyzdTrmZ47-5_6uPGQhzNzzMViPBHNg6yfUnNVpvw&seq=1). In the United States, *Salmonella* Heidelberg ranks as the [third most common](https://pubmed.ncbi.nlm.nih.gov/18757574/) serotype found in retail meat and food animal isolates. It is also among the[ top four serotypes ](https://journals.asm.org/doi/10.1128/aac.01333-10)isolated from blood samples. Notably, 12%–13% of *Salmonella* Heidelberg infections in [North America](https://wwwnc.cdc.gov/eid/article/16/1/09-0729_article) result in invasive disease, a rate significantly higher than the U.S. average of 7% for all *Salmonella* infections.


## Genetic Characteristics

Serovar Heidelberg has been found to be [monophyletic](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full). Previously, a [phage typing scheme](http://jcm.asm.org/content/41/9/4279.full) was developed for serovar Heidelberg in Canada; this scheme recognizes 49 phage types. *S.* Heidelberg accounts for the [8.6% of ceftriaxone-resistant](https://www.cdc.gov/salmonella/pdf/heidelberg-508c.pdf) *Salmonella* isolated from humans in the U.S. [SARA33](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3790100/) is a multidrug resistant (MDR) strain. This strain was found, *in silico*, to contain a novel integron cassette; in addition, it was found to carry the following resistance genes: *aac*(*6′*)*-ly*, *aadA5*, *aadB*, *aa*(*6′*)*-33*, and *aadA1*, *sul1* and *sul2*, *bla<sub>OXA-2</sub>* and *bla<sub>TEM</sub>*, *tetD*. Resistance genes in *S.* Heidelberg have been identified in the chromosome (resistant islands) and in plasmids. Different incompatibility types (A/C, FIB, HI2) have been identified in the [plasmids](http://www.ncbi.nlm.nih.gov/pubmed/23251446) that carried resistance genes in *S.* Heidelberg. A [MDR S. Heidelberg strain](https://archive.cdc.gov/#/details?url=https://www.cdc.gov/salmonella/heidelberg-10-13/index.html) (resistant to ampicillin, chloramphenicol, gentamicin, kanamycin, streptomycin, sulfisoxazole, and tetracycline) was found to cause a multistate outbreak linked to the Foster Farms brand chicken. [Etter et al.](https://journals.asm.org/doi/10.1128/aem.01065-19) analyzed a total of nine Heidelberg isolates collected during the outbreak investigation and revealed that (i) six *Salmonella* Heidelberg isolates linked to the foodborne outbreak exhibited increased heat tolerance, (ii) one of these heat-tolerant isolates also demonstrated enhanced biofilm formation under stressful conditions, (iii) heat stress triggered higher expression of multidrug efflux and virulence genes in *Salmonella* Heidelberg, and (iv) the outbreak-related isolates appeared to be transcriptionally prepared to better withstand processing stresses and potentially to cause infection.

Phylogenetic and pan-genomic analyses that included two of the sequenced *S.* Heidelberg strains and other *Salmonella* serovars concluded that *S.* Heidelberg (i) belongs to clade A, according to [den Bakker et al. (2011)](http://www.biomedcentral.com/1471-2164/12/425) classification, (ii) appears to have a single evolutionary origin, and (iii) contains 74 [gene families](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3175032/) unique to this serovar.


## Animal Reservoir

[Poultry](<https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1282832/pdf#:~:text=Salmonella%20enterica%20subspecies%20enterica%20serovar%20Heidelberg%20(Salmonella%20Heidelberg)%20is%20primarily,Clothier%20and%20Byrne%2C%202016>).), including chickens and turkeys, are the main reservoirs for serovar Heidelberg. It has also been found in [other animals such as cattle, swine, and dogs](<https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Heidelberg%22)>).


## Geographical Distribution

Serovar Heidelberg has been reported [worldwide](<https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(taxgroup_name:%22Salmonella%20enterica%22)%20AND%20computed_types:(%22serotype=Heidelberg%22)>), mainly in [North America](https://pmc.ncbi.nlm.nih.gov/articles/PMC3768931/).


## Human/Animal Outbreaks

Multiple Heidelberg outbreaks have been linked to poultry, most of them in the US. Here are some example:


## Recalls

<sup>1</sup> Tyson Foods, Inc. issued a recall of around 33,840 pounds of mechanically separated chicken products due to potential contamination with *Salmonella *Heidelberg. These products were manufactured on October 11, 2013, and distributed exclusively for institutional use across the nation.


## References

- https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=About%20a%20new%20type%20of%20bacteria%20form%20the%20paratyphoid%20enteritis%20group&author=VH%20Habbs&publication_year=1933&pages=367-374&

- https://www.jstor.org/stable/1588278?casa_token=sTneApEGY84AAAAA%3AxCcrraoy6aPu-M6ejq3MtxjBB1MMgP_Z6YwUi4_cVoYN9dHewNuMwxG9EeHAe6ua_dq1zAdT8vyzdTrmZ47-5_6uPGQhzNzzMViPBHNg6yfUnNVpvw&seq=1

- https://pubmed.ncbi.nlm.nih.gov/18757574/

- https://journals.asm.org/doi/10.1128/aac.01333-10

- https://wwwnc.cdc.gov/eid/article/16/1/09-0729_article

- https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2025.1547190/full

- https://journals.asm.org/doi/full/10.1128/jcm.41.9.4279-4284.2003

- https://www.cdc.gov/salmonella/pdf/heidelberg-508c.pdf

- https://pmc.ncbi.nlm.nih.gov/articles/PMC3790100/

- https://pubmed.ncbi.nlm.nih.gov/23251446/

- https://journals.asm.org/doi/10.1128/aem.01065-19

- https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-425

- https://pmc.ncbi.nlm.nih.gov/articles/PMC3175032/

- https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1282832/pdf#:~:text=Salmonella%20enterica%20subspecies%20enterica%20serovar%20Heidelberg%20(Salmonella%20Heidelberg)%20is%20primarily,Clothier%20and%20Byrne%2C%202016).

- https://pmc.ncbi.nlm.nih.gov/articles/PMC3768931/

- https://pubmed.ncbi.nlm.nih.gov/34932458/#:~:text=We%20report%20a%20multistate%20Salmonella,indicators%20of%20severity%20and%20invasiveness.

- https://archive.cdc.gov/#/details?url=https://www.cdc.gov/salmonella/heidelberg-11-16/index.html

- https://archive.cdc.gov/www_cdc_gov/salmonella/heidelberg-10-13/index.html

- https://archive.cdc.gov/www_cdc_gov/salmonella/2011/chicken-liver-1-11-2012.html

- https://pubmed.ncbi.nlm.nih.gov/25865382/

- https://pmc.ncbi.nlm.nih.gov/articles/PMC9151100/

- https://www.canada.ca/en/public-health/services/reports-publications/canada-communicable-disease-report-ccdr/monthly-issue/2005-31/restaurant-foodhandler-associated-outbreak-salmonella-heidelberg-gastroenteritis-identified-calls-local-telehealth-service-edmonton.html

- https://webgate.ec.europa.eu/rasff-window/screen/notification/680242

- https://webgate.ec.europa.eu/rasff-window/screen/notification/679279

- https://webgate.ec.europa.eu/rasff-window/screen/notification/609282

- https://archive.cdc.gov/www_cdc_gov/salmonella/heidelberg-01-14/advice-institutions.html
