# *S.* Kentucky

!!! info "At a Glance"

    | | |
    |---|---|
    | **Antigenic Formula** | `8,20:i:z6` |
    | **NCBI Pathogen Detection** | [View isolates](https://www.ncbi.nlm.nih.gov/pathogens/isolates/) |


## Background Information

*Salmonella enterica* subsp. *enterica* serovar Kentucky (antigenic formula 8,20:​i:z<sub>6</sub>) is a serovar of the O:8 (C<sub>2</sub>-C<sub>3</sub>) serogroup. In [North America](https://pmc.ncbi.nlm.nih.gov/articles/PMC6606869/), serovar Kentucky remained the number one most prevalent serovar in poultry products. This serovar is commonly found in food animals in the U.S. (chickens, and cattle), but [rarely](https://www.frontiersin.org/journals/sustainable-food-systems/articles/10.3389/fsufs.2021.695368/full) is reported in human cases. It is the most common serovar identified in non-clinical non-human sources, according to [CDC, 2011](https://www.cdc.gov/salmonella/pdf/salmonella-atlas-508c.pdf). However, in Europe, serovar Kentucky is found in approx. 1% of the human cases ([top 10](https://www.sciencedirect.com/science/article/abs/pii/S0963996911002869)). A number of human cases have been reported in Europe in [travelers from Asia and Africa](https://efsa.onlinelibrary.wiley.com/doi/full/10.2903/j.efsa.2018.5500). However, there are limited outbreaks with epidemiological investigation and clear source identification have been reported.


## Genetic Characteristics

Serovar Kentucky str. [CVM29188](https://pmc.ncbi.nlm.nih.gov/articles/PMC2747853/) is a multidrug resistant strain that contains three plasmids of 46, 101, and 146 kb. The two large plasmids are conjugative plasmids that carry antimicrobial resistance genes. The largest plasmid of 146 kb shows the same backbone and virulence genes as virulence plasmids of Avian Pathogenic *E. coli* (APEC). The [SGI1-K](https://pmc.ncbi.nlm.nih.gov/articles/PMC1797678/) strain, isolated from Indian spices, is another multidrug resistant strain that contains the resistance genes inserted in the chromosome in a genomic island (*Salmonella* Genomic Island 1 variant SGI1-K). In [Europe](https://pubmed.ncbi.nlm.nih.gov/21813512/) and [Canada](http://wwwnc.cdc.gov/eid/article/19/6/12-1351_article.htm), ciprofloxacin-resistant serovar Kentucky strains have been found. A highly resistant clone of *S.* Kentucky [(MLST type ST198)](http://www.thelancet.com/journals/laninf/article/PIIS1473-3099%2813%2970124-5/abstract), has been reported to be isolated in Canada and in Europe in travelers returning from Asia and Africa. This strain shows resistance to β-lactam antibiotics, carbapenems, quinolones, aminoglycosides, co-trimoxazole (trimethoprim-sulfamethoxazole), and to azithromycin. Molecular characterization shows a chromosomal [genomic island](http://aac.asm.org/content/56/10/5096.full) carrying the resistance genes that confer resistance to these antimicrobials.

Kentucky [ST 152 ](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001089#tab2)was found to be predominant in US chickens and appears to show attenuated human virulence. Of 5151 genomes [Richards et al.](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001089#tab2) analyzed, 90.3 % were isolated from broilers, while 5.9 %, 3.0% were from humans and cattle, respectively. Notably, two serovar Kentucky-II isolates demonstrated an average 8.6-fold higher replication rate in macrophages compared to Kentucky-I isolates. Phylogenetic and pan-genomic analyses that included completely sequenced *S.* Kentucky and other *Salmonella* serovars concluded that *S.* Kentucky (i) belongs to clade A, according to [den Bakker et al. (2011)](http://www.biomedcentral.com/1471-2164/12/425) classification, (ii) appears to have several evolutionary origin, and (iii) contains 127 [gene families](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3175032/) unique to this serovar. [Chen et al.](https://journals.asm.org/doi/10.1128/mbio.01913-24?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed#T4) found that serovar Kentucky is a polyphyletic serovar with two lineages identified (Kentucky A and Kentucky B) and two stand-alone singletons that do not cluster with any other *Salmonella* Kentucky isolates. While Kentucky A is associated with chicken (the majority of isolates are from the US), Kentucky B (the majority of isolates are from the UK) is associated with cattle. There are two clonal subclades identified in the Kentucky B Clade 2.


## Animal Reservoir

[Chicken](https://journals.asm.org/doi/10.1128/mmbr.00015-13) is the main reservoir for serovar Kentucky. However, one clade has been found to be associated with [cattle](https://journals.asm.org/doi/10.1128/mbio.01913-24?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed).


## Geographical Distribution

Serovar Kentucky is widely distributed and can be found [globally](https://pubmed.ncbi.nlm.nih.gov/21813512/).


## Human/Animal Outbreaks

Human outbreaks associated with serovar Kentucky are rare. Here we described one outbreak we found.

<sup>1</sup> 25 other cases associated with this outbreak (n=26) were infected by *Salmonella *Muenchen.

Animal outbreaks:

<sup>1</sup> Three of the 7 long-nosed fur seals showed clinical disease and died within 3 days.


## Border Rejections

<sup>1</sup> *Salmonella* Kentucky and *Salmonella* Senftenberg were both found.

<sup>2</sup> *Salmonella* Kentucky and *Salmonella* Senftenberg were both found.


## References

- [https://pmc.ncbi.nlm.nih.gov/articles/PMC6606869/](https://pmc.ncbi.nlm.nih.gov/articles/PMC6606869/#:~:text=Typhimurium%20is%20the%20most%20common,common%20serovar%2C%20with%20Typhimurium%20second)

- https://www.frontiersin.org/journals/sustainable-food-systems/articles/10.3389/fsufs.2021.695368/full

- https://www.cdc.gov/salmonella/pdf/salmonella-atlas-508c.pdf

- https://www.sciencedirect.com/science/article/abs/pii/S0963996911002869

- https://efsa.onlinelibrary.wiley.com/doi/full/10.2903/j.efsa.2018.5500

- https://pmc.ncbi.nlm.nih.gov/articles/PMC2747853/

- https://pmc.ncbi.nlm.nih.gov/articles/PMC1797678/

- https://pubmed.ncbi.nlm.nih.gov/21813512/

- https://wwwnc.cdc.gov/eid/article/19/6/12-1351_article#r3

- https://www.thelancet.com/journals/laninf/article/PIIS1473-3099%2813%2970124-5/abstract

- https://journals.asm.org/doi/full/10.1128/aac.00732-12

- https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001089#tab2

- https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-425

- https://pmc.ncbi.nlm.nih.gov/articles/PMC3175032/

- https://journals.asm.org/doi/10.1128/mmbr.00015-13

- https://journals.asm.org/doi/10.1128/mbio.01913-24?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed

- https://pubmed.ncbi.nlm.nih.gov/21813512/

- https://archive.cdc.gov/#/details?url=https://www.cdc.gov/salmonella/muenchen-02-16/index.html

- https://pubmed.ncbi.nlm.nih.gov/33269723/

- https://webgate.ec.europa.eu/rasff-window/screen/notification/591197

- https://webgate.ec.europa.eu/rasff-window/screen/notification/607336

- https://webgate.ec.europa.eu/rasff-window/screen/notification/416870

- https://webgate.ec.europa.eu/rasff-window/screen/notification/494162
